src/plugins_types.c

```diff
@@ -54,15 +54,14 @@ ly_err_msg_create(LY_ERR *err_code, LY_VECODE err_vecode, const char *err_msg, .
     struct ly_err_item *ret;
     char *msg;
     int  print_ret;
-
     va_list(print_args);
 
     if (err_msg != NULL) {
         va_start(print_args, err_msg);
         print_ret = vasprintf(&msg, err_msg, print_args);
     }
 
-    if ((print_ret < 0) || (err_msg == NULL)) {
```

Comment:
I guess this has nothing to do with the subject. It is another bug - the `print_ret` is uninitiated when `err_msg` is not NULL.